### ISSUE TITLE
Add community channel, logos, repo in owl file

### ISSUE THREAD

**Issue by bryan-brancotte** [edamontology/edamontology#682]
Following a suggestion made in https://github.com/edamontology/edam-browser/issues/77, adding link to the gitter, the logo, the repo, ... could help setting up automatically an instance for https://github.com/edamontology/edam-bioimaging, but also building the interface of the edam-browser more generically.

```xml
        <doap:Version rdf:datatype="http://www.w3.org/2001/XMLSchema#decimal">1.25</doap:Version>
        <foaf:page rdf:resource="http://edamontology.org/page"/>
        <???:gitter rdf:resource="https://gitter.im/edamontology/edamontology"/>
        <???:logo rdf:resource="https://github.com/edamontology/edamontology/blob/main/EDAM-logo-square.svg"/>
        <???:source rdf:resource="https://github.com/edamontology/edamontology/"/>
```

**Comment by matuskalas** [comment]
This will indeed be very nice to add!👍🏽

@albangaignard would you please advise on which are the most standard Semantic Web attributes for a **source code repository** and a **logo**? Should we look in Wikipedia|DbPedia, or where? And maybe also the **communication channel (mailing list, chat)**, but that we could perhaps also leave out at first.

**Comment by drashti171299** [comment]
Yes @matuskalas. Do let me know which attributes should I add in owl file.

**Comment by bryan-brancotte** [comment]
Thanks for the annotations, and https://lov.linkeddata.es !

**Comment by matuskalas** [comment]
Indeed, thank you so much!🙏🏽

I realised that for the source code repo we could also use the one defined in EDAM itself (used for formats): `edam:repository`

**Comment by matuskalas** [comment]
This is a side track, but related: @albangaignard do you know what the way is to get the attributes & properties defined in EDAM listed in https://lov.linkeddata.es? Things like the `repository`, `has_input`, `has_output`, `has_function` might be quite broadly usable...
(Sorry for the side track, we should move it to a separate issue if a discussion starts)

**Comment by matuskalas** [comment]
Back on track: @drashti171299, please add `repository` and `foaf:logo` for now. Many thanks! 🙏🏽

**Comment by drashti171299** [comment]
ok @matuskalas .
